.github/workflows/gitlab-runners-ubi8-git-pr.yaml

```diff
@@ -0,0 +1,26 @@
+name: ubi8-git-pr-ci
+on:
+  pull_request:
+    paths:
+      - gitlab-runners/ubi8-git/**
+jobs:
+  build:
+    env:
+      context: gitlab-runners/ubi8-git
+      image_name: gitlab-runners-ubi8-git
+    # Only build on redhat-cop repo, do not attempt to build on forks
+    if: github.repository == 'redhat-cop/containers-quickstarts'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Check and verify version.json
+        id: check_version
+        run: |
+          echo -n ::set-output name=IMAGE_TAGS::
+          echo $(jq -r '.version' ${context}/version.json)
+      - name: Build image and test image
+        run: |
+          echo "Running: docker build -f ${context}/Dockerfile -t ${image_name}:${{ steps.check_version.outputs.IMAGE_TAGS }} ${context}"
+          docker build -f ${context}/Dockerfile -t ${image_name}:${{ steps.check_version.outputs.IMAGE_TAGS }} ${context}
```

Comment:
any reason not to use an action here to do the building? i.e.: https://www.docker.com/blog/first-docker-github-action-is-here/

Reply:
No reason, changed to `docker/build-push-action`